Makefile

```diff
@@ -25,20 +25,20 @@ unregister-full:
 
 # only aware of aula sources
 sensei: .phony aula.unregister
-	$(EXEC) sensei $(AULA_SOURCES) tests/Spec.hs $(SENSEI_DEFAULT_ARGS) $(SENSEI_ARGS)
+	$(EXEC) sensei -j5 $(AULA_SOURCES) tests/Spec.hs $(SENSEI_DEFAULT_ARGS) $(SENSEI_ARGS)
```

Comment:
> Can you point me to the documentation for '-j5'?

what about `--ghc-options="-Werror -Wall -O0"` then?  or `docker exec -it`?  `grep -Hni`?  i think the documentation of the clis should not be part of the Makefile.